rules_building_block/persistence_ld_preload_execution.toml

```diff
@@ -0,0 +1,135 @@
+[metadata]
+bypass_bbr_timing = true
+creation_date = "2024/06/03"
+integration = ["endpoint"]
+maturity = "production"
+updated_date = "2024/06/03"
+
+[rule]
+author = ["Elastic"]
+building_block_type = "default"
+description = """
+This rule monitors for the execution of a Linux process with the `LD_PRELOAD` or `LD_LIBRARY_PATH` environment variable
+set. `LD_PRELOAD` and `LD_LIBRARY_PATH` are environment variables that can be used to load shared libraries before the
+standard libraries. Attackers can abuse these environment variables to load malicious libraries and execute arbitrary
+code. This rule helps to detect potential attempts to hijack the execution flow of a process, which can be used for
+defense evasion, persistence, and privilege escalation.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.process*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Process Execution with LD_Preload or LD_Library_Path Environment Variable Set"
```

Comment:
since the rule scope is limited to descendant of 3 processes sshd,console and terminal, the rule name and description should be adjusted to align with rule scope.